File: pythonCodewars/Get the mean of an array.py
# DESCRIPTION:
# It's the academic year's end, fateful moment of your school report. The averages must be calculated. All the students come to you and entreat you to calculate their average for them. Easy ! You just need to write a script.

# Return the average of the given array rounded down to its nearest integer.

# The array will never be empty.

def get_average(marks):
    # Calculate the sum of all marks
    total_sum = sum(marks)
    
    # Calculate the number of marks
    count = len(marks)
    
    # Calculate the average and round down by using integer division
    average = total_sum // count
    
    return average

File: c#/Get the mean of an array.cs
// It's the academic year's end, fateful moment of your school report. The averages must be calculated. All the students come to you and entreat you to calculate their average for them. Easy ! You just need to write a script.

// Return the average of the given array rounded down to its nearest integer.

// The array will never be empty.

// using System.Linq;
// using System.Collections.Generic;
// using System;

using System.Linq;
using System.Collections.Generic;
using System;

public class Kata
{
  public static int GetAverage(int[] marks)
  {
    // Sum all the marks and divide by the number of marks to get the average
    return marks.Sum() / marks.Length;
  }
}